spec/openapi.yaml

```diff
@@ -3288,6 +3288,10 @@ components:
     OrderDebtor:
       description: Order debitor details
       properties:
+        id:
+          description: "ID of logged in customer, if any"
```

Comment:
The customer isn't necessarily logged-in. We match by email for orders made via the API.

Reply:
```suggestion
          description: "ID of the corresponding customer, if any"
```